VitalTrackApp/src/screens/Analytics.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {View, Text, ScrollView, StyleSheet, Image} from 'react-native';
+
+export default function AnalyticsScreen() {
+  return(
+    <ScrollView>
+      <View style={styles.container}>
+        <Text style={styles.title}>Analytics Page</Text>
+        <Text style={styles.subtitle}>An overview of your wellbeing</Text>
+
+        <View style={styles.section}>
+            <Text style={styles.title}>Insights</Text>
+            <Text style={styles.subtitle}>You felt [less stressed] when you [did yoga] this week.</Text>
+        </View>
+
+        <View style={styles.section}>
+            <Text style={styles.title}>Mental Health</Text>
+            <Text style={styles.caption}>Your mood this week</Text>
+            <Image
+                source={require('../../assets/images/moodgraph-example.png')}
+                style={styles.graph}
+            />
+        </View>
+
+        <View style={styles.section}>
```

Comment:
This would be a good spot to make the get_tips API call and display the trained LLM response